### ISSUE TITLE
Line numbers and segments in code examples

### ISSUE THREAD

**Issue by TelegramSam** [decentralized-identity/waci-presentation-exchange#35]
All examples should use the line-numbered format for code. Below each code block, we should indicate which portions of the example are from which different specs and formats.

**Comment by brentzundel** [comment]
and each example should be complete, per the call 20210614

**Comment by bumblefudge** [comment]
I check the numbering thing and spec-up doesn't support the 
```json=
notation that some markdown systems do (and which github doesn't). happy to add the numbers and links manually once the examples are complete!

**Comment by bumblefudge** [comment]
On today's Interop call, Pam recommended:
- color coding (maybe correspond to the image at the top from #42)
- JWT.IO -style parser/linter ? cut and paste the example into such a thing for further guidance?

**Comment by OR13** [comment]
@csuwildcat this could be your moment.

**Comment by csuwildcat** [comment]
@OR13 can add things - what is the full list of asks? Is this the correct set of items:

1. Line numbers
2. Click-to-copy code/examples
3. Segments in code examples (what does this mean?)

**Comment by OR13** [comment]
Blocked by https://github.com/decentralized-identity/spec-up/issues/24 Won't fix... feel free to open a PR when it is supported.
